Sources/NiceComponents/Title/SectionTitle.swift

```diff
@@ -24,7 +24,12 @@ public struct SectionTitle: NiceText {
     public var body: some View {
         Text(text)
             .foregroundColor(style.color)
-            .scaledFont(name: style.theme.name, size: style.theme.size, weight: style.theme.weight)
+            .scaledFont(
+                name: style.theme.name,
+                size: style.theme.size,
+                weight: style.theme.weight,
+                maxSize: style.theme.dynamicTypeMaxSize
+            )
             .fixedSize(horizontal: false, vertical: true)
```

Comment:
Not sure, but we should verify if fixed vertical size breaks dynamic type sizing in any way

Reply:
I checked the example screen at each version of dynamic type sizing and it doesn't seem to.